pkg/installmanager/installmanager_test.go

```diff
@@ -693,3 +694,37 @@ func TestInstallManagerSSH(t *testing.T) {
 		})
 	}
 }
+
+func TestIsBootstrapComplete(t *testing.T) {
+	cases := []struct {
+		name             string
+		errCode          int
+		expectedComplete bool
+	}{
+		{
+			name:             "complete",
+			errCode:          0,
+			expectedComplete: true,
+		},
+		{
+			name:             "not complete",
+			errCode:          1,
+			expectedComplete: false,
+		},
+	}
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			dir, err := ioutil.TempDir("", "TestIsBootstrapComplete")
```

Comment:
Looks good just need to defer a cleanup on this dir.

Reply:
Ah, yes. Good catch.